addons/viewdistance/functions/fnc_adaptViewDistance.sqf

```diff
@@ -22,8 +22,10 @@ if (!GVAR(enabled) || isNull ACE_player) exitWith {};
 
 private _vehicle = vehicle ACE_player;
 
-private _landVehicle = _vehicle isKindOf "LandVehicle" || {_vehicle isKindOf "Ship_F"};
-private _airVehicle = _vehicle isKindOf "Air";
+private _isControllingDrone = !( (UAVControl (getconnectedUAV _vehicle) select 1) isEqualTo "" );
```

Comment:
I think this should be `getconnectedUAV ACE_player` in all 3 instances.
If player is riding in a vehicle while controlling a UAV `getconnectedUAV vehicle player` will return `objNull`